ui/00_message/master/area1_spot_info.xml

```diff
@@ -6,7 +6,7 @@
 ことから、全ての民にとって寄り所と言える地域に
 なった。そのため、比較的魔物の勢力は弱いが
 オークなど強力な魔物の侵入は脅威となる</original>
-    <translation>A large plain spread across central Lestania.
+  <translation>A large plain spread across central Lestania.
```

Comment:
I apologize, when I said two spaces, I meant two spaces instead of using a tab, not only two spaces. Like this line would need four spaces.